src/test/java/pl/patrycja/ox/gameflow/ModeTest.java

```diff
@@ -0,0 +1,38 @@
+package pl.patrycja.ox.gameflow;
+
+import org.mockito.Mockito;
+import org.testng.annotations.Test;
+import pl.patrycja.ox.Player;
+import pl.patrycja.ox.Sign;
+import pl.patrycja.ox.ui.UI;
+
+import java.util.List;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+
+public class ModeTest {
+
+    UI ui = Mockito.mock(UI.class);
+
+    @Test
+    public void askWhichPlayerShouldStart() {
+        //given
+        Sign signX = Sign.X;
+        Sign signO = Sign.O;
+        Mode mode = new AutomaticTests(ui);
+        String get_initial_sign = "get_initial_sign";
```

Comment:
Cooooo to za nazwa XD ? czemu łamiemy konwencje camelcase'a ?

Reply:
poprawie, nie zauwazylam, bo byla wygenerowana automatycznie :P